lib/administrate/resource_resolver.rb

```diff
@@ -22,6 +20,10 @@ def resource_title
       model_path_parts.join(" ")
     end
 
+    def namespace
+      controller_path.split("/").first
```

Comment:
This is where we calculate the namespace. In the default setup, this returns the string `"admin"`.